backend/src/Manager/Admin/Order/AdminOrderManager.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Manager\Admin\Order;
+
+use App\Repository\OrderEntityRepository;
+
+class AdminOrderManager
+{
+    private OrderEntityRepository $orderEntityRepository;
+
+    public function __construct(OrderEntityRepository $orderEntityRepository)
+    {
+        $this->orderEntityRepository = $orderEntityRepository;
+    }
+
+    public function getOrdersByStateForAdmin(string $state): string
+    {
+        return $this->orderEntityRepository->getOrdersByStateForAdmin($state);
+    }
+
+    public function getAllOrdersCountForAdmin(): string
+    {
+        return $this->orderEntityRepository->getAllOrdersCountForAdmin();
+    }
+}
```

Comment:
missing the empty line after class end

Reply:
GitHub doesn't show the empty line after the closing bracket of the class.

Reply:
though, please check in your class